app/src/main/res/values/strings.xml

```diff
@@ -829,6 +829,19 @@
         <item>BH1750</item>
     </string-array>
 
+    <string name="show_guide_text">Show Guide</string>
+    <string name="hide_guide_text">Hide Guide</string>
+
+    <string name="wave_gen_guide_intro">The wave generator can be used to generate different types of waves like
+        sine wave, square wave, saw tooth wave and also allow us to changes their characteristics like frequency, phase, duty.
+        \nIt also allows us to produce PWM signals having different phase and duty.</string>
+
+    <string name="wave_gen_guide_desc">To generate sine wave or saw tooth wave:\n\n
+        \u2022 Connect the pins Wave pins (S1,S2) to the channel pins CH1, CH2 as shown in the figure\n\n
+        \u2022 Select the Wave1 button for S1 pin and Wave2 button for S2 pin\n\n
+        \u2022 Press sin image button for Sine wave and saw tooth image button for Saw Tooth wave\n\n
```

Comment:
Is it sin or sine? Also you frequently change the case. Please make it uniform. It is scientific document.